### ISSUE TITLE
Outdated version on modernizr.com/download

### ISSUE THREAD

**Issue by dmke** [Modernizr/Modernizr#2592]
As the title said, the website creates a build with Modernizr v3.6.0.

**Comment by Markel** [comment]
Yep, has been like that for a long time, sorry for the inconvenience. If I'm not wrong the problem resides on the need for a redeployment, but @patrickkettner may correct me. Moreover, he probably will be able to give you a better answer than me.

As my own suggestion (have been thinking about it for a while now) maybe the server should retrieve the necessary Modernizr files from the Github API in a way that the Modernizr version auto-updates. Although this is probably more an issue for the [modernizr-neue repo](https://github.com/Modernizr/modernizr-neue)
